### PR TITLE
Embeddings pipeline

### PR DESCRIPTION
**Pull request by Ilyes-Azni**
Built the logic for the embedding plus I had some tests running and the model works fine! But I need some further help and review from y'all, because I am so stuck in putting the pipeline to work. The logic works on an independent notebook for instance but i can't make it work when launching the command _kedro run --pipeline [pl_name]_

### CODE REVIEW

File: goodreads-predictor/src/goodreads_predictor/pipelines/word_embeddings/nodes.py [hunk 1]
```diff
@@ -1,4 +1,12 @@
...
-generated using Kedro 0.19.6
-"""
+# nodes.py
+
+import pandas as pd
+from sentence_transformers import SentenceTransformer
+
+def create_embeddings(books_loaded):
+    model = SentenceTransformer('all-MiniLM-L6-v2')
+    df = pd.read_parquet(books_loaded)
```
Comment (Ilyes-Azni) [inline goodreads-predictor/src/goodreads_predictor/pipelines/word_embeddings/nodes.py:8]: **I think that the problem might come from there**

Comment (Egjfour) [reply goodreads-predictor/src/goodreads_predictor/pipelines/word_embeddings/nodes.py:8]: Yep! The read_parquet is being handled natively by Kedro. You don't need to do any input/output for your functions. Just specify them in the catalog.yml file

Comment (Egjfour) [reply goodreads-predictor/src/goodreads_predictor/pipelines/word_embeddings/nodes.py]: Updated with commits below

**Comment by Egjfour** [comment]
Added in web scraping and combined these branches to remove merge conflicts with main. Please make sure to call "git pull" to bring in changes before working on new branches/pushing branches

**Review by Egjfour — APPROVED** [review]

**Pull request merged**